src/awkward/operations/ak_from_jax.py

```diff
@@ -1,52 +0,0 @@
-# BSD 3-Clause License; see https://github.com/scikit-hep/awkward/blob/main/LICENSE
-
-from __future__ import annotations
-
-from awkward import jax
-from awkward._dispatch import high_level_function
-from awkward._layout import from_arraylib, wrap_layout
-
-__all__ = ("from_jax",)
-
-
-@high_level_function()
-def from_jax(
```

Comment:
I don't think we should get rid of this function. We still want to support converting a jax array to an awkward array with e.g. cpu backend, we're just getting rid of jax internally in awkward.

Reply:
Thanks, I agree. I was debating if it is useful to keep these high level functions.

Reply:
restored!

Reply:
This and the `ak.to_jax` implementation do not need the jax backend/nplike. We can restore their behavior without reintroducing all of that code.